app/src/server/emails/templates/ProjectInvitation.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { Button, EmailLayout, Heading, Text } from "./layout";
+
+interface Props {
+  subject: string;
+  projectName: string;
+  invitationSenderName: string;
+  invitationSenderEmail: string;
+  invitationLink: string;
+}
+
+const ProjectInvitation = ({
+  subject,
```

Comment:
We are using it in the `<Heading>{subject}</Heading>`

Reply:
Ah I see that now. Do we need to accept it as a prop, or can we just hardcode it in?

Reply:
We were using the same text in a function that sends this email. But it's okay to remove it. I do not pass subjects in other templates because the text is a bit different.